src/com/axis/rtspclient/FLVMux.as

```diff
@@ -130,8 +131,25 @@ package com.axis.rtspclient {
         var bit_depth_chroma_minus8:uint = sps.readUnsignedExpGolomb();
         var qpprime_y_zero_transform_bypass_flag:uint = sps.readBits(1);
         var seq_scaling_matrix_present_flag:uint = sps.readBits(1);
-        if (1 === seq_scaling_matrix_present_flag) {
-          ErrorManager.dispatchError(822, null, true);
+
+        if (seq_scaling_matrix_present_flag) {
+          var i:uint  =0;
+          for (i = 0; i < 8; i++) {
+            var seq_scaling_list_present_flag:uint = sps.readBits(1);
+            if (seq_scaling_list_present_flag) {
+              var sizeOfScalingList:uint = (i < 6) ? 16 : 64;
+              var lastScale:uint = 8;
+              var nextScale:uint = 8;
+              var j:uint = 0;
+              for (j = 0; j < sizeOfScalingList; j++) {
+                if (nextScale != 0) {
+                  var delta_scale:uint = sps.readSignedExpGolomb();
+                  nextScale = (lastScale + delta_scale + 256) % 256;
+                }
+                lastScale = (nextScale == 0) ? lastScale : nextScale;
+              }
+            }
+          }
```

Comment:
This doesn't seem quite right. See [here for the algorithm](http://stackoverflow.com/questions/6394874/fetching-the-dimensions-of-a-h264video-stream), section 7.3.2.1.1.

If `seq_scaling_matrix_present_flag` then it should loop 8 times if `chroma_format_idc` is anything other than 3, otherwise it should loop 12 times.